glances/plugins/containers/model.py

```diff
@@ -297,6 +307,8 @@ def msg_curse(self, args=None, max_width=None):
         ret.append(self.curse_add_line(msg))
         msg = ' {:<7}'.format('Tx/s')
         ret.append(self.curse_add_line(msg))
+        msg = '{:<30}'.format('Socket URL')
```

Comment:
We also should display this only optionally. As in the user can opt in if they want this to be displayed.

The containers plugin has very little screen room, so this will reduce it even further